lua/octo/menu.lua

```diff
@@ -58,6 +58,8 @@ local function get_filter(opts, kind)
       val = json.stringify(val)
       val = string.gsub(val, '"OPEN"', "OPEN")
       val = string.gsub(val, '"CLOSED"', "CLOSED")
+      val = string.gsub(val, '"MERGED"', "MERGED")
+      val = string.gsub(val, '"ALL"', "ALL")
```

Comment:
test

comment at commit level (line 62) that will be deleted in commit afterwards